launchcontainer/tests.py

```diff
@@ -5,25 +5,19 @@
 import mock
 import unittest
 
+from django.conf import settings
+
 from xblock.field_data import DictFieldData
 from opaque_keys.edx.locations import Location, SlashSeparatedCourseKey
 
-from django.test.utils import override_settings
-
-from .launchcontainer import API_URL_DEFAULT
-
+from django.test import override_settings
 
-API_URL_CONF_SET_WITH_GOOD_ORG = {
-    "org": "https://api.org.com"
-}
+from .launchcontainer import DEFAULT_WHARF_URL
 
 
-API_URL_CONF_SET_WITH_BAD_ORG = {
-    "other_org": "https://api.foo.com",
-    "default": "https://api.default.com"
-}
-
-API_URL_CONF_NOT_SET = {}
+WHARF_ENDPOINT_GOOD = "https://api.org.com"
```

Comment:
I know this is only for testing but can we use e.g. `api.localhost`, just in case?